storage_service/administration/urls.py

```diff
@@ -10,17 +9,15 @@
     path("version/", views.version_view, name="version"),
     path("users/", views.user_list, name="user_list"),
     path("users/create/", views.user_create, name="user_create"),
-    re_path(r"^users/(?P<id>[-\w]+)/edit/$", views.user_edit, name="user_edit"),
-    re_path(r"^users/(?P<id>[-\w]+)/$", views.user_detail, name="user_detail"),
+    path("users/<int:id>/edit/", views.user_edit, name="user_edit"),
+    path("users/<int:id>/", views.user_detail, name="user_detail"),
     path("language/", views.change_language, name="change_language"),
     path("keys/", views.key_list, name="key_list"),
-    re_path(
-        r"^keys/(?P<key_fingerprint>[\w]+)/detail$", views.key_detail, name="key_detail"
-    ),
+    path("keys/<str:key_fingerprint>/detail/", views.key_detail, name="key_detail"),
```

Comment:
Why do we need a new trailing slash in this new route?

Reply:
oops!